packages/opencensus-core/README.md

```diff
@@ -1,11 +1,74 @@
 # OpenCensus Core Node.js
-[![Gitter chat][gitter-image]][gitter-url]
+[![Gitter chat][gitter-image]][gitter-url] ![Node Version][node-img] [![NPM Published Version][npm-img]][npm-url] ![dependencies Status][dependencies-status] [![Downloads Stats][npm-downloads]][npm-url] ![Apache License][license-image]
 
-OpenCensus for Node.js is an implementation of OpenCensus, a toolkit for collecting application performance and behavior monitoring data. 
+OpenCensus for Node.js is an implementation of OpenCensus, a toolkit for collecting application performance and behavior monitoring data. It currently includes 3 apis: stats, tracing and tags.
 
 The library is in alpha stage and the API is subject to change.
 
-Please join [gitter](https://gitter.im/census-instrumentation/Lobby) for help or feedback on this project.
+## Installation
+
+Install the opencensus-core package with NPM:
+```bash
+npm install @opencensus/core
+```
+
+## Usage
+
+#### Set up a new Stats manager instance.
+
+To enable metrics, we’ll import a few items from OpenCensus Core package.
+
+```javascript
+const { Stats, MeasureUnit, AggregationType } = require('@opencensus/core');
+
+// Create the Stats manager
+const stats = new Stats();
+
+// The latency in milliseconds
+const mLatencyMs = stats.createMeasureDouble(
+  "repl/latency",
+  MeasureUnit.MS,
+  "The latency in milliseconds"
+);
+```
+
+#### Create Views and Tags:
+
+We now determine how our metrics will be organized by creating ```Views```. We will also create the variable needed to add extra text meta-data to our metrics – ```methodTagKey```, ```statusTagKey```, and ```errorTagKey```.
+
+```javascript
+const methodTagKey = "method";
+const statusTagKey = "status";
+const errorTagKey = "error";
+
+const latencyView = stats.createView(
+  "demo/latency",
+  mLatencyMs,
+  AggregationType.DISTRIBUTION,
+  [methodTagKey, statusTagKey, errorTagKey],
+  "The distribution of the latencies",
+  // Bucket Boundaries:
+  // [>=0ms, >=25ms, >=50ms, >=75ms, >=100ms, >=200ms, >=400ms, >=600ms, >=800ms, >=1s, >=2s, >=4s, >=6s]
+  [0, 25, 50, 75, 100, 200, 400, 600, 800, 1000, 2000, 4000, 6000]
+);
+```
+
+#### Recording Metrics:
+
+Now we will record the desired metrics. To do so, we will use ```stats.record()``` and pass in measurements.
+
+```javascript
+const endTime = new Date();
+const tags = {method: "repl", status: "OK"};
+
+stats.record({
+  measure: mLatencyMs,
+  tags,
+  value: (new Date()) - startTime.getTime()
```

Comment:
Its not clear how is `startTime` defined

Reply:
Done.